src/types/balance.ts

```diff
@@ -0,0 +1,18 @@
+export type AccountBalance = {
+  hntBalance: number
+  stakedHntBalance: number
+  iotBalance: number
```

Comment:
Balances being a number is going to be a problem

Reply:
javascript float will be destroyed

Reply:
These are integer values, so I think we're fine. I updated to bigint for good measure, but if I remember right App Center doesn't let you use them. We will see...